libraries/c_sdk/standard/https/src/private/iot_https_internal.h

```diff
@@ -184,13 +184,43 @@
 #ifndef IOT_HTTPS_MAX_ALPN_PROTOCOLS_LENGTH
     #define IOT_HTTPS_MAX_ALPN_PROTOCOLS_LENGTH    ( 255 ) /* The maximum alpn protocols length is chosen arbitrarily. */
 #endif
-/* Provide the User-Agent Value definition from coreHTTP. */
+#ifndef IOT_HTTPS_QUEUE_RECV_TICKS
+    #define IOT_HTTPS_QUEUE_RECV_TICKS             pdMS_TO_TICKS( 200U ) /* The ticks to wait for a #xQueueReceive to complete. */
+#endif
+#ifndef IOT_HTTPS_QUEUE_SEND_TICKS
+    #define IOT_HTTPS_QUEUE_SEND_TICKS             portMAX_DELAY /* The ticks to wait for a #xQueueSendToBack to complete. */
```

Comment:
I dont think that this should be the max by default because I don't think the orignal function IotTaskPool_Schedule() blocked at all.

Reply:
Thanks for catching. I actually switched these up. It is the RECV_TICKS that should block indefinitely (as we do not want to allocate CPU time when there are no requests to process), and I will be setting this to SEND_TICKS to 0 to emulate the original non-blocking behavior.